runeq/resources/common.py

```diff
@@ -141,19 +144,42 @@ def get(self, id: str) -> ItemBase:
             f"{self._item_class.__name__} with ID {id}"
         )
 
-    def add(self, *items: ItemBase):
+    def add(self, items: Union[ItemBase, Iterable[ItemBase]]):
```

Comment:
I'd consider instead creating a separate `update()` method when trying to add a set of items, just so we stick with more pythonic syntax for sets. Update also is a little more telling that an existing item with the same key can be overwritten.

Reply:
🤦‍♀️ aaaand I read your other comment first. but I had the same thought! I will make the change